test/e2e/framework/nodes_util.go

```diff
@@ -40,10 +40,10 @@ func EtcdUpgrade(target_storage, target_version string) error {
 	}
 }
 
-func IngressUpgrade() error {
+func IngressUpgrade(mode bool) error {
```

Comment:
minor: naming a boolean `mode` sounds a bit odd...Probably `isDowngrade`?

Reply:
Done, made it isUpgrade.